utils/output_manager.py

```
@@ -0,0 +1,129 @@
+        for msg in recent_messages:
+            if msg['role'] == 'user':
+                self._format_user_content(msg['content'])
+            elif msg['role'] == 'assistant':
+                self._format_assistant_content(msg['content'])
+
+    def _format_user_content(self, content: Any):
+        """Format and display user content."""
+        if isinstance(content, list):
+            for content_block in content:
+                if isinstance(content_block, dict):
+                    if content_block.get("type") == "tool_result":
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)